setup.cfg

```diff
@@ -37,7 +37,6 @@ install_requires =
 
 [options.extras_require]
 tests =
-    invenio-app>=1.4.0
     pytest-black-ng>=0.4.0
     pytest-cache>=1.0
     pytest-runner>=2.6.2
```

Comment:
should the section `[options.entry_points]` be removed?

Reply:
Yes! Nice catch :)